circus-train-s3-s3-copier/src/main/java/com/hotels/bdp/circustrain/s3s3copier/aws/TransferManagerFactory.java

```diff
@@ -15,8 +15,15 @@
  */
 package com.hotels.bdp.circustrain.s3s3copier.aws;
 
+
+import java.util.concurrent.Executors;
+
+import org.slf4j.Logger;
+import org.slf4j.LoggerFactory;
 import org.springframework.stereotype.Component;
 
+import jline.internal.Log;
```

Comment:
This doesn't look right?

Reply:
looks like left over from an incorrect import. May need to tweak my IDE settings 😅 
Removed.